### PR TITLE
Add python plugin and update all other python packages

### PR DESCRIPTION
**Pull request by keilr**
This PR adds the pulp python plugin to the "all in one" container image and updates all other python packages.

### CODE REVIEW

**Comment by keilr** [comment]
cc @dkliban @gerrod3

File: pulp_fedora31/Containerfile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   pulp-ansible==${PULP_ANSIBLE_VERSION} \
+  pulp-container==${PULP_CONTAINER_VERSION} \
+  pulp-file==${PULP_FILE_VERSION} \
+  pulp-maven==${PULP_MAVEN_VERSION} \
+  pulp-python==${PULP_PYTHON_VERSION} \
   pulp-rpm==${PULP_RPM_VERSION} \
-  pulp-maven==${PULP_MAVEN_VERSION}
 
-RUN ln /usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_ansible.conf
-RUN ln /usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_container.conf
```
Comment (mdellweg) [inline pulp_fedora31/Containerfile:20]: Those lines are most definitely needed.

Comment (keilr) [reply pulp_fedora31/Containerfile:20]: Hi @mdellweg thanks for your answer. How can I fix this? Were the files renamed?
```
ERROR: Invalid requirement: '/usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf'
Hint: It looks like a path. File '/usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf' does not exist.

ERROR: Invalid requirement: '/usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf'
Hint: It looks like a path. File '/usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf' does not exist.
```

Comment (mdellweg) [reply pulp_fedora31/Containerfile:20]: I don't know. Did the pip install path change?

Their path in the package looks right:
https://github.com/pulp/pulp_container/blob/master/MANIFEST.in#L2

Comment (keilr) [reply pulp_fedora31/Containerfile]: fixed. I forgot to remove the backslash on the previous RUN statement...

**Comment by dkliban** [comment]
We will be adding Python plugin when it has a GA release (soon).

**Pull request closed without merging**